src/main/java/org/jenkinsci/test/acceptance/po/WizardCustomizeJenkins.java

```diff
@@ -0,0 +1,74 @@
+package org.jenkinsci.test.acceptance.po;
+
+import static org.jenkinsci.test.acceptance.Matchers.hasContent;
+
+import java.util.concurrent.Callable;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.NoSuchFrameException;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Page object for Wizard Customize Jenkins Page.
+ *
+ */
+public class WizardCustomizeJenkins extends PageObject {
+
+    public WizardCustomizeJenkins(Jenkins jenkins) {
+        super(jenkins.injector, jenkins.url(""));
+    }
+
+    public WizardCustomizeJenkins doInstallSuggested() {
+        By suggestedButtonLocator = By.partialLinkText("Install suggested plugins");
+        waitFor(suggestedButtonLocator, 30);
+        Control installSuggested = control(suggestedButtonLocator);
+        installSuggested.click();
+        return this;
+    }
+
+    public WizardCustomizeJenkins doSelectPluginsToInstall() {
+        By selectPluginsLocator = By.partialLinkText("Select plugins to install");
+        waitFor(selectPluginsLocator, 30);
+        Control selectPlugins = control(selectPluginsLocator);
+        selectPlugins.click();
+        waitFor(hasContent("Plugin Manager"));
+        return this;
+    }
+
+    @SuppressWarnings("deprecation") // FIXME: change the detection method when we have id or name in the frame.
+    public void shouldFinishInstallSuccessfully() {
+        waitForCond(new Callable<Boolean>() {
+            @Override
+            public Boolean call() throws Exception {
+                try {
+                    WebElement element = driver.switchTo().defaultContent().switchTo().frame(1).findElement(By.name("username"));
```

Comment:
Because `by.name` has been created explicitly to be used against the jenkins ui, and if for some reason something changes in the UI that brokes the name convention is likely that `by.name` will be modified accordingly. Relying on `By` means that your new tests will have to be fixed particularly, so IMO is a litle more robust to use `by.name`